core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -145,19 +145,31 @@ class AdminClient(val time: Time,
 
   def describeConsumerGroup(groupId: String): Option[List[ConsumerSummary]] = {
     val group = describeGroup(groupId)
-    if (group.state == "Dead")
-      return None
+
+    group.state match {
+      case "Dead" =>
+        println(s"Consumer group `${groupId}` does not exist.")
```

Comment:
@ewencp Thanks a lot for your feedback. There are some restructuring of the code to `ConsumerGroupCommand.scala` in the [PR for KAFKA-3144](https://github.com/apache/kafka/pull/1336) as suggested by @hachikuji, that would affect this PR. It would be great if we could finalize that first and then rebase and continue with this PR. Please advise if you disagree. Thanks.